assets/site/guide.sass

```diff
@@ -142,15 +142,13 @@ html.cockpit-guide
 
   h2, h3, h4, h5, h6,
   dl.toc .part,
-  dl.toc .chapter,
     position: static
     padding: 1rem 0 0.5rem
 
-  h2,
-  dl.toc .part
-    font-size: 1.72rem
+  dl.toc .chapter
+    position: static
 
-  h3,
+  h3
```

Comment:
Neither this.

Reply:
I should have made a different commit from my understanding , but this is just a trailing comma, it's no error but it's unnecessary.

Reply:
I moved this to a separate commit.